examples/bridge-app/linux/main.cpp

```diff
@@ -325,25 +319,6 @@ EmberAfStatus HandleWriteOnOffAttribute(Device * dev, chip::AttributeId attribut
     return EMBER_ZCL_STATUS_SUCCESS;
 }
```

Comment:
Can't comment on the text above, but _I think_ you should be able to delete `DECLARE_DYNAMIC_ATTRIBUTE_LIST_BEGIN(descriptorAttrs)` as well as the instantiation of `descriptorAttrs` on your dynamic endpoint.

Reply:
If I delete the DECLARE_DYNAMIC_ATTRIBUTE_LIST_BEGIN(descriptorAttrs), the dynamic endpoint will not contain the descriptor cluster.